org.upscayl.app.yml

```diff
@@ -0,0 +1,68 @@
+id: io.github.upscayl.Upscayl
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: upscayl-run
+separate-locales: false
+
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

Comment:
```suggestion
```

I'll assume this doesn't support wayland.

Reply:
It does support wayland, one person created an issue for it as well.

Reply:
then it `--socket=fallback-x11` on line 14.

also `--share=ipc` is missing, it's needed either.

Reply:
> then it `--socket=fallback-x11` on line 14.

It doesn't use the Wayland backend by default. This needs to be explicitly run with launch flags.

Reply:
then `--socket=x11` and no wayland